### ISSUE TITLE
Make CSV_EXPORT utilize the ert-api

### ISSUE THREAD

**Issue by oysteoh** [equinor/ert#3423]
This issue was originally part of #2189, but decided to split it out due to unclear implications to our users. Not all data needed for the export is part of our storage and will imply changes in the way we store data / configure ERT to internalize etc. Thus I created a new epic which can be prioritized and picked up again at a later point in time.

- Make the job CSV_EXPORT that generate data of similar format as CSV_EXPORT2. 
- Backend of the `export` button in ERT should be changed from semeio job to CSV_EXPORT
- All data should be retrieved through the ert-api
- Code related to the export should be added into the ERT repository.

**Comment by oyvindeide** [comment]
Blocked by the design matrix not being implemented.
